src/game/puppycam2.c

```diff
@@ -478,6 +475,78 @@ void puppycam_reset_values(void) {
     gPuppyCam.splineProgress = 0;
 }
 
+static void create_puppycam1_nodes(void) {
+    u32 i, flagsAdd, flagsRemove;
+    Vec3s pos, diff;
+    struct sPuppyAngles *tempAngle;
+    for (i = 0; i < sizeof(newcam_fixedcam) / sizeof(struct newcam_hardpos); i++) {
+        if (newcam_fixedcam[i].newcam_hard_levelID != gCurrLevelNum || newcam_fixedcam[i].newcam_hard_areaID != gCurrAreaIndex)
+            continue;
+        if ((sPuppyVolumeStack[gPuppyVolumeCount] = mem_pool_alloc(gPuppyMemoryPool, sizeof(struct sPuppyVolume))) == NULL) {
+            gPuppyError |= PUPPY_ERROR_POOL_FULL;
+        #if PUPPYPRINT_DEBUG
+                append_puppyprint_log("Puppycamera volume allocation failed.");
```

Comment:
Maybe this has been addressed in Puppyprint already, but `append_puppyprint_log`'s actual definition should be ifdef'd, not the calls to it.

Reply:
I'll leave this as is for now, and make this change in Puppyprint 2.1.

Reply:
This was not addressed in Puppyprint 2.1; please fix.

Reply:
This conditional check should be removed now.

Also I just noticed `#if`. Double check and make sure those don't exist anywhere else in here.